### ISSUE TITLE
Theme select dropdown options hidden behind header

### ISSUE THREAD

**Issue by Akszyn** [home-assistant/frontend#9512]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

in my case, the service that changes the appearance, who is called frontend.set_theme does nothing despite pressing the call service button

### Describe the behavior you expected

After calling the frontend.set_theme service and entering any name of your theme, it is set, no matter what theme we have currently selected

### Steps to reproduce the issue

1.Change to any theme
2.copying the name of another theme
3.Go to the developer tools tab and then services
4. Find the service responsible for changing the theme
5.Entering a different name of the themes from the list that is different than the current one
6. No reaction to the theme change and no log entries about it 
7. Logout and logon required 
8. Successfully repeating step 3 and 4

### What version of Home Assistant Core has the issue?

core-2021.6.7

### In which browser are you experiencing the issue with?

The latest version of the Chrome Browser (91.0.4472.124)

### Which operating system are you using to run this browser?

The latest version of the Windows 10 Pro 21H1 (19043.1081)

**Comment by bramkragten** [comment]
You can override the backend theme on the frontend. The service sets the backend theme, so if the frontend is not set to `Backend theme` it will ignore the backend theme setting.

**Comment by tomlut** [comment]
Hi Bram, unfortunately this was a poorly described issue. 

The actual issue is that the "Backend-selected" option disappears from the option list (possibly hidden under the header bar) after selecting an alternate theme. See:

![dc9206b1bff60e931d8d80b3f980aad3b9959edb](https://user-images.githubusercontent.com/10679300/124616399-633a1900-deb9-11eb-9301-e08321eacb22.jpeg)

I have been unable to reproduce this as I only have a short list of themes, lending weight to the possibility that the option is hidden under the title bar. 

Others have noticed the same thing for some time:

https://community.home-assistant.io/t/theme-backend-selected-out-of-reach/43948

**Comment by Akszyn** [comment]
the mentioned theme appears only after the page is reduced on the computer as can be seen in the attached picture![Przechwytywanie](https://user-images.githubusercontent.com/57450821/124618364-036f4e00-de78-11eb-81a9-2442cbc0e2d7.JPG)

**Comment by tomlut** [comment]
Yeah, so the list needs to be brought to the front of the title bar. 

If Bram does not reopen this you may need to open a new issue. I suggest you leave out everything you wrote originally and concentrate on the "Backend-selected" option being hidden under the title bar when the list of themes is long. Use your screen shots. and the link I posted above.

**Comment by Mariusthvdb** [comment]
this could be related? https://github.com/home-assistant/frontend/issues/9416

**Comment by SgtBatten** [comment]
Seeing this issue myself. If I use a theme with a transparent header I can see the backend option but can't press it still

**Comment by spacegaier** [comment]
I cannot reproduce this issue in dev with neither Firefox nor Chrome (on desktop at least). The long dropdown menus always get a scrollbar here so I can access all entries without issue.

**Comment by Mariusthvdb** [comment]
can confirm this to work as expected I Chrome on a Mac. Safari however shows no scrollbar, nor can be scrolled. as mentioned in https://github.com/home-assistant/frontend/issues/9512#issuecomment-874963376

**Comment by SgtBatten** [comment]
I have confirmed that the location of the themes setting on your screen before opening the dropdown makes all the difference. 

If I position themes towards the top half of my screen before opening the drop down I can see backend selected. If I position theme in the bottom half of my screen then it is hidden after opening the drop down.

**Comment by Mariusthvdb** [comment]
still happening (on Safari)

**Comment by Mariusthvdb** [comment]
Is fixed so could be closed
